iampolicymanagementv1/iam_policy_management_v1_examples_test.go

```diff
@@ -742,6 +753,76 @@ var _ = Describe(`IamPolicyManagementV1 Examples Tests`, func() {
 			Expect(response.StatusCode).To(Equal(201))
 			Expect(policyTemplate).ToNot(BeNil())
 		})
+
+		It(`CreatePolicyS2STemplate request example`, func() {
+			fmt.Println("\nCreatePolicyS2STemplate() result:")
+			// begin-create_policy_template
```

Comment:
This has the same name as line 708. Can be an issue with example generation in the docs if this are named same? I think there might be a collison.

Reply:
yes, only one code block per operation